lib/logic/models/sort.dart

```diff
@@ -292,13 +271,8 @@ class AlbumSort extends Sort<Album> {
     );
   }
 
-  @override
-  AlbumSort get withDefaultOrder {
-    return copyWith(orderAscending: feature.defaultOrderAscending);
-  }
-
   int _fallbackYear(Album a, Album b) {
-    return a.year.compareTo(b.year);
+    return (a.year ?? 0).compareTo((b.year ?? 0)); // TODO: Decide how to sort null values
```

Comment:
I left this as a todo for a later PR, but we should discuss how we sort null values. I would say they should always be at the end, no matter what (ignoring ascending/descending). What's your opinion on this?

<details><summary>I have some nice extension methods that will make it easy that I'll add in another PR.</summary>

```dart
extension NullableCompare<T> on T? {
  int compareTo(T? other) {
    final self = this;
    if (self != null) {
      self.compareToNullable(other);
    }
    if (other != null) {
      return -other.compareToNullable(self);
    }
    return 0;
  }
}

extension CompareNullable<T> on dynamic {
  int compareToNullable(T? other) {
    if (other != null) {
      return compareTo(other);
    }
    return -1;
  }
}
```
</details>

Reply:
SGTM